src/components/AccountList.tsx

```diff
@@ -129,9 +135,7 @@ function AccountItem({
             <Text emoji style={[a.font_bold]}>
               {profile?.displayName || account.handle}{' '}
             </Text>
-            <Text emoji style={[t.atoms.text_contrast_medium]}>
-              {account.handle}
-            </Text>
+            <Text style={[t.atoms.text_contrast_medium]}>{account.handle}</Text>
```

Comment:
Don't need to sanitize the display name here since it's the user's own account, but the `sanitizeHandle` util is used to insert the warning icon for invalid handles, and if that happens we need the `emoji` prop. Pry good to add that sanitization for consistency, idk why it's not here already.

Reply:
Ah ok good to know